.github/workflows/openms_ci_matrix_full.yml

```diff
@@ -361,6 +361,15 @@ jobs:
           ${{ github.workspace }}/OpenMS/bld/*.exe
           ${{ github.workspace }}/OpenMS/bld/*.deb
           ${{ github.workspace }}/OpenMS/bld/*.pkg
+
+    - if: steps.set-vars.outputs.pkg_type != 'none' && [[ "${{ matrix.os }}" == ubuntu-* ]] # Only upload docs when we are building the package, use the ubuntu build simply 'cause its fast
```

Comment:
@jpfeuffer this seems to trigger on both Ubuntu runners, unlike uploading the installers. Am I missing something in the syntax of the if statement?

Reply:
Yep. See the suggestion that we merged. You don't need the square brackets in if fields.

Reply:
Thanks, I saw that your update fixed it. I'm currently trying to figure out why the html/ directory doesn't build.